server/constants/constants.go

```diff
@@ -0,0 +1,14 @@
+package constants
+
+const (
+	SERVER_ADDRESS string = ":8000"
+	NETWORK_TCP string = "tcp"
+)
+
+const (
+	HTTP_METHOD_GET = "GET"
+)
+
+const (
+	GITHUB_TOKEN = "GITHUB_TOKEN"
```

Comment:
I would follow convention and use a more explicit name for readability.

```suggestion
	GithubTokenEnvName = "GITHUB_TOKEN"
```